README.md

```diff
@@ -105,6 +105,45 @@ e.g. `helm -n  laa-fala-staging delete el-123-fee-change`
 It is also possible to manually deploy to an environment from the command line, the structure of the command can be found in `bin/deploy.sh`
 
 
+## CHECK referrals
+
+As part of improvement works to 'Check if you can get legal aid and Find a legal advisor' (CHECK) in February 2025, a new set of pages & urls were created that would allow users to be sent from CHECK to FALA. This allowed the CLA team (who were maintaining CHECK at the time) to then remove a set of redundant ‘mini-FALA’ pages. Epic for the work can be viewed [here](https://dsdmoj.atlassian.net/browse/EL-1891).
```

Comment:
i think we should remove the text about CLA team and mini-fala, we just want to cover the `what` here not the `why`